src/app/models/question-dto.ts

```diff
@@ -0,0 +1,74 @@
+export class QuestionDto {
```

Comment:
This is good for now but in the long run, we should turn this into a library component that both of the UI codebases share.